src/ast/nodes/ForInStatement.ts

```diff
@@ -49,11 +49,11 @@ export default class ForInStatement extends StatementBase {
 
 	include(context: InclusionContext, includeChildrenRecursively: IncludeChildren) {
 		this.included = true;
-		this.left.includeAllDeclaredVariables(context, includeChildrenRecursively);
+		this.left.include(context, includeChildrenRecursively || true);
```

Comment:
How is it different from `this.left.include(context, true);`?

Reply:
There is a special string value for `includeChildrenRecursively` that is like `true` but also includes "related" code. Something we use to deoptimize try-catch statements.

Reply:
You would need to do something rather nonsensical like this to get a change from that:
https://rollupjs.org/repl/?version=2.38.4&shareable=JTdCJTIybW9kdWxlcyUyMiUzQSU1QiU3QiUyMm5hbWUlMjIlM0ElMjJtYWluLmpzJTIyJTJDJTIyY29kZSUyMiUzQSUyMmZ1bmN0aW9uJTIwdGVzdChjYWxsYmFjayklMjAlN0IlNUNuJTIwJTIwdHJ5JTIwJTdCJTVDbiUyMCUyMCUyMCUyMGZvciUyMCglN0Jmb28lMjAlM0QlMjBjYWxsYmFjaygpJTdEJTIwaW4lMjAlN0JiYXIlM0ElMjAxJTdEKSUyMCU3QiU3RCUzQiU1Q24lMjAlMjAlN0QlMjBjYXRjaCUyMCU3QiU3RCU1Q24lN0QlNUNuJTVDbnRlc3QoKCklMjAlM0QlM0UlMjAlN0IlNUNuJTVDdCUyRiUyRiUyMFRoaXMlMjBpcyUyMG9ubHklMjByZXRhaW5lZCUyMGJlY2F1c2UlMjBvZiUyMHRoZSUyMHNwZWNpYWwlMjBwYXJhbWV0ZXIlMjB2YWx1ZSU1Q24lMjAlMjBPYmplY3QuY3JlYXRlKG51bGwpJTVDbiU3RCklM0IlMjIlMkMlMjJpc0VudHJ5JTIyJTNBdHJ1ZSU3RCU1RCUyQyUyMm9wdGlvbnMlMjIlM0ElN0IlMjJmb3JtYXQlMjIlM0ElMjJlcyUyMiUyQyUyMm5hbWUlMjIlM0ElMjJteUJ1bmRsZSUyMiUyQyUyMmFtZCUyMiUzQSU3QiUyMmlkJTIyJTNBJTIyJTIyJTdEJTJDJTIyZ2xvYmFscyUyMiUzQSU3QiU3RCU3RCUyQyUyMmV4YW1wbGUlMjIlM0FudWxsJTdE